test/api.js

```diff
@@ -357,6 +357,50 @@ tape('findAndTombstone and tombstoning branchStream', (t) => {
   })
 })
 
+tape('findOrCreate() recps', (t) => {
+  const boxdir = '/tmp/metafeeds-metafeed-box2'
+  const boxKey = ssbKeys.loadOrCreateSync(path.join(dir, 'secret'))
```

Comment:
Is it on purpose that the path arg uses `dir` not `boxdir`?

Reply:
No, that's a bug. Will fix